WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -532,6 +538,22 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                     AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATION_PENDING_DRAFTS_SETTINGS_DISABLED);
                 }
             }
+        } else if (key.equals(getString(R.string.wp_pref_custom_notification_sound))) {
+            final String defaultPath =
+                    getString(R.string.notification_settings_item_sights_and_sounds_choose_sound_default);
+            final String value = sharedPreferences.getString(key, defaultPath);
+
+            if (value.trim().toLowerCase().startsWith("file://")) {
+                // sound path begins with 'file://` which will lead to FileUriExposedException when used. Revert to
+                //  default and let the user know.
```

Comment:
super small nit, a double space seems to be here before the word `default`

Reply:
That's by design! That's how I prefer to wrap long comment lines :)